src/helpers.ts

```diff
@@ -5,7 +5,7 @@ export const addPathToUrl = (customNetworkUrl: string, linkType: string, suffixT
 
   const auth = username ? `${username}:${password}` : '';
 
-  const parsedUrl = new URL(`${protocol}//${auth}${host}${newPath}${search}${hash}`);
+  const parsedUrl = new URL(`${protocol}//${auth}${host}${hash.endsWith('#/') ? '#' : hash}${newPath}${search}`);
```

Comment:
A small nitpick, if all you want to do here is take off the trailing slash from `hash`, it feels like you could do this more succinctly. What do you think about this?

``` suggestion
  const parsedUrl = new URL(`${protocol}//${auth}${host}${hash.replace(/\/$/, '')}${newPath}${search}`);
```